content/transfer-apps.md

```diff
@@ -12,7 +12,7 @@ You can deploy an application to one Galaxy account at a time. Galaxy utilizes t
 
 If you are a member of two or more accounts, you can then transfer the application to a different account.
 
-<h2 id="meteor-13">Transfer with Meteor 1.3</h2>
+<h2 id="meteor-13">Transfer with Meteor > 1.3</h2>
```

Comment:
"with Meteor 1.3 or higher versions"

And use the same language for "with Meteor 1.2 or lower versions"